openverse-api/catalog/api/serializers/provider_serializers.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework import serializers
+
+from catalog.api.models import ContentProvider, SourceLogo
+
+
+class ProviderSerializer(serializers.ModelSerializer):
+    source_name = serializers.CharField(source='provider_identifier')
+    display_name = serializers.CharField(source='provider_name')
+    source_url = serializers.URLField(source='domain_name')
```

Comment:
Would be nice to add an example for `domain_name` (is it `https://www.wikimedia.org/` for wikimedia, or something else?)

Reply:
Added examples in b93edeb.